source/java/hdf/hdf5lib/H5.java

```diff
@@ -235,10 +237,22 @@ public synchronized static void loadH5Lib() {
         {
             return;
         }
-        if (NativeLibraryUtilities.loadNativeLibrary("jhdf5") == false)
+        if (System.getProperty("native.caching.libpath") == null)
+        {
+            try {
+				System.setProperty("native.caching.libpath", Files.createTempDirectory("jhdf5").toString());
```

Comment:
@ctruden Setting `native.caching.libpath` to a common temporary directory may be an alternative to using `Fiji.app/lib`.